BlueprintUICommonControls/Sources/AttributedLabel.swift

```diff
@@ -156,6 +170,7 @@ extension AttributedLabel {
                 .attributedText
                 .applyingDefaultFont()
                 .replacingLinkAttributes()
+                .normalizingLineBreakMode(for: model.numberOfLines)
```

Comment:
This is minor, but I wonder how much of a perf hit we get here with the 4 copy, edit, returns from this builder style.

Reply:
Yeah, I think I'll combine them. Probably not the best thing to do for every label on every render.